packages/cli/src/commands/init/printRunInstructions.js

```diff
@@ -9,19 +9,22 @@
  */
 
 import path from 'path';
+import fs from 'fs-extra';
 import chalk from 'chalk';
 import {logger} from '@react-native-community/cli-tools';
 
-function printRunInstructions(projectDir: string, projectName: string) {
+async function printRunInstructions(projectDir: string, projectName: string) {
   const absoluteProjectDir = path.resolve(projectDir);
-  const xcodeProjectPath = `${path.resolve(
-    projectDir,
-    'ios',
-    projectName,
-  )}.xcodeproj`;
+  const iosProjectDir = path.resolve(projectDir, 'ios');
+
+  const iosPodsFile = path.resolve(iosProjectDir, `${projectName}.xcworkspace`);
+  const isUsingPods = await fs.pathExists(iosPodsFile);
+
   const relativeXcodeProjectPath = path.relative(
     process.cwd(),
-    xcodeProjectPath,
+    isUsingPods
```

Comment:
@grabbou: it's good to check this out as well, I assume it's fine but it's better to be sure.

I noticed it yesterday that I was assuming that it was `.xcworkspace` without verifying first.